great_expectations/datasource/data_connector/file_path_data_connector.py

```diff
@@ -259,6 +263,10 @@ def _generate_batch_spec_parameters_from_batch_definition(
 
     def _validate_batch_request(self, batch_request: BatchRequestBase):
         super()._validate_batch_request(batch_request=batch_request)
+
+        # <WILL> how do we check that : has batch_request_base.data_asset_name????
+        # this is
```

Comment:
Is this comment still supposed to be here? Having some trouble understanding it.

Reply:
oop sorry. this should have been deleted